lib/Sabberworm/CSS/CSSList/CSSList.php

```diff
@@ -246,11 +246,18 @@ public function splice($iOffset, $iLength = null, $mReplacement = null)
 
     /**
      * Removes an item from the CSS list.
-     * @param RuleSet|Import|Charset|CSSList $oItemToRemove May be a RuleSet (most likely a DeclarationBlock), a Import, a Charset or another CSSList (most likely a MediaQuery)
+     *
+     * @param RuleSet|Import|Charset|CSSList|int $oItemToRemove May be a RuleSet (most likely a DeclarationBlock), a Import, a Charset or another CSSList (most likely a MediaQuery)
```

Comment:
I strongly recommend not mixing types likes this in a parameter. Instead, I propose a new, separate method `removeByKey(int $key)`.

Reply:
Agreed.

Reply:
Happy to make the change but are the points raised in https://github.com/sabberworm/PHP-CSS-Parser/pull/224#issuecomment-841852668 still a concern with this PR?